ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -292,11 +363,14 @@ class AdminWorkspaceImpl extends React.Component<UrlParamsProps, State> {
           <h3>Cloud Storage Objects</h3>
           <div className='cloud-storage-objects' style={{marginTop: '1rem'}}>
             {workspaceInfoField('GCS bucket path', resources.cloudStorage.storageBucketPath)}
-            {workspaceInfoField('# of Notebook Files', resources.cloudStorage.notebookFileCount)}
```

Comment:
strictly speaking, this is the # of notebook files in the notebook dir.  Rogue / user-created notebook files can reside elsewhere!